recipes/pyamrex/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "pyamrex" %}
+{% set version = "23.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AMReX-Codes/pyamrex/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 44dd3ab467260ea797dec4fcb939599fad811e8ac598bdc0ab05da24369606fa
+  patches:
+    # Adds pip_install_nodeps target, part of 23.11+
+    #   https://github.com/AMReX-Codes/pyamrex/pull/203
+    - 203.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang         # [win]
```

Comment:
I am not sure we support this. Any reason you can't use the default compilers?

Reply:
Thanks for checking. Yes, this is the recommended way to get modern OpenMP to work on Windows according to @isuruf and works quite well on existing feedstocks.

Reply:
Thanks for the info!